src/averageAges.js

```diff
@@ -15,6 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const men
+    = arguments.length > 1
+      ? people.filter(person => Math.ceil(person.died / 100) === century
+        && person.sex === 'm')
+      : people.filter(person => person.sex === 'm');
+
+  return men.reduce(
+    (prev, curr) => prev + (curr.died - curr.born), 0
+  ) / men.length;
```

Comment:
You'll use same logic in next function, better to move it to separate helper function